.travis.yml

```diff
@@ -2,10 +2,18 @@ language: c++
 sudo: false
 env:
   - NODE_VERSION="0.10"
+  - NODE_VERSION="0.11"
   - NODE_VERSION="0.12"
+  - NODE_VERSION="iojs-v1"
```

Comment:
@nfischer: I know `v0.11` is supposed to be the same as `v0.12`, but I feel like it's probably worth testing.

As for iojs, yes, node >`4` are actually iojs releases. So the release line looks like this(-ish):

```
v0.9 --- v0.10 ---- v0.11 --- v0.12 ---- v0.13 --- v0.14 --- v4.0 --- v5.0 --- [Future]
               \                                           /
                 ------ v1.0 ------ v2.0 ------ v3.0 -----
```